src/terra-dev-site/doc/application/how-to.2/theme-strategy.doc.mdx

```diff
@@ -0,0 +1,166 @@
+
+`example-component/ExampleComponent.module.scss`
+
+```scss
+// The values in this file represent the default theme. The default theme is applied using the default values in the var() function.
+:local {
+  .example-component {
+    background-color: var(--terra-application-example-component-background-color, #fff);
+    color: var(--terra-application-example-component-color, inherit);
+    font-size: var(--terra-application-example-component-font-size, 1rem);
+  }
+}
+```
+
+To add themes, create new files for each theme and include them in the original SCSS file. Theme files define variables that are applied when the theme becomes active. Themes must be locally scoped and must use a supported theme class name.
+
+`example-component/ExampleComponent.orion-fusion-theme.module.scss`
+
+```scss
+:local {
+  .orion-fusion-theme {
+    --terra-application-example-component-background-color: #f6f8fa;
+    --terra-application-example-component-color: rgb(36, 41, 46);
+    --terra-application-example-component-font-size: 2rem;
+  }
+}
+```
+
+`example-component/ExampleComponent.clinical-lowlight-theme.module.scss`
+
+```scss
+:local {
+  .clinical-lowlight-theme {
+    --terra-application-example-component-background-color: rgb(34, 42, 46);
+    --terra-application-example-component-color: rgb(178, 181, 182);
+    --terra-application-example-component-font-size: 1rem;
+  }
+}
+```
+
+Include the new theme files in the original SCSS file.
+
+`example-component/ExampleComponent.module.scss`
+
+```scss
+// Import each theme.
+@import './ExampleComponent.clinical-lowlight-theme.module';
+@import './ExampleComponent.orion-fusion-theme.module';
+
+:local {
+  .example-component {
+    background-color: var(--terra-application-example-component-background-color, #fff);
+    color: var(--terra-application-example-component-color, inherit);
+    font-size: var(--terra-application-example-component-font-size, 1rem);
+  }
+}
+```
+
+The recommended file structure ends up looking like this:
+
+```
+.
+├── example-component
+│   ├── ExampleComponent.clinical-lowlight-theme.module.scss
+│   ├── ExampleComponent.jsx
+│   ├── ExampleComponent.module.scss
+│   └── ExampleComponent.orion-fusion-theme.module.scss
+```
+
+## Webpack
+
+To enable themes, teams must define a terra-theme.config.js configuration file. This file is read by the [terra-toolkit postcss loader](https://github.com/cerner/terra-toolkit/tree/master/config/webpack/postcss) to aggregate themes. This file must be defined for theme CSS assets to be included in the webpack bundle. The loader runs automatically for webpack configurations using terra-toolkit's webpack config.
```

Comment:
```suggestion
To enable themes, teams must define a `terra-theme.config.js` configuration file. This file is read by the [terra-toolkit postcss loader](https://github.com/cerner/terra-toolkit/tree/master/config/webpack/postcss) to aggregate themes. This file must be defined for theme CSS assets to be included in the webpack bundle. The loader runs automatically for webpack configurations using terra-toolkit's webpack config.
```

Reply:
should we link the default webpack config?

Reply:
Added a link to the default webpack config: https://github.com/cerner/terra-application/pull/25/commits/2a93997b6888d4c01d6be725178806bdd92ac386